src/sentry/monkey/pickle.py

```diff
@@ -208,8 +118,5 @@ def py3_compat_pickle_loads(*args, **kwargs):
             return original_pickle_loads(*args, **kwargs)
 
     pickle.load = py3_compat_pickle_load
-    pickle.dump = py3_compat_pickle_dump
     pickle.loads = py3_compat_pickle_loads
```

Comment:
there's still a lot of hits on the `loads` metric -- going to add sentry instrumentation after this PR to understand what the actual error is here (the comments mention something about `datetime` but I cannot reproduce any of those findings)